src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -393,4 +468,23 @@ export default class EnhancedReactTable<T> extends React.Component<IEnhancedReac
             ...this.state
         });
     }
+
+    private handleChangeItemsPerPage(itemsPerPage:number) {
+        const nextState = Object.assign({}, this.state);
```

Comment:
is this to get around ts issue with partial setState?  if so, i would rather we use the TS hack for this.  casting the object as the type of the state.